src/GovUk.Education.ExploreEducationStatistics.Data.Services/Mappings/DataServiceMappingProfiles.cs

```diff
@@ -33,12 +33,12 @@ public DataServiceMappingProfiles()
                     }
                 });
 
-            CreateMap<Publication, TopicPublicationViewModel>();
+            CreateMap<Content.Model.Publication, TopicPublicationViewModel>();
 
-            CreateMap<Theme, ThemeViewModel>()
+            CreateMap<Content.Model.Theme, ThemeViewModel>()
                 .ForMember(dest => dest.Topics, opts => opts.Ignore());
 
-            CreateMap<Topic, TopicViewModel>()
+            CreateMap<Content.Model.Topic, TopicViewModel>()
```

Comment:
See later comment about ThemeService that shouldn't exist in the Data.Services anymore, so if these view models and mappings are still needed, they should be in the Content.Api.

Reply:
Created a subtask to address this: EES-2520.